components/rowListScene.brs

```diff
@@ -1,12 +1,21 @@
 sub init()
```

Comment:
The logic in this file is complex. Please simplify it.

Reply:
Thanks so much ..
I'll be waiting